cmd/plugin/node_server.go

```diff
@@ -47,6 +47,7 @@ func NewNodeServer(driver *csicommon.CSIDriver, mounter backend.Mounter, imageSv
 			SecretStore:        secretStore,
 			Mounter:            mounter,
 		}),
+		k8smounter: k8smount.New(""),
```

Comment:
FYI. this is needed, so that the "unsafe mount" detection happens only once. It clutters the logs quite a bit